distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Entity_Naming_Properties.enso

```diff
@@ -33,11 +33,14 @@ new jdbc_connection =
         if column_limit == 0 then
             Panic.throw (Illegal_State.Error "Unexpected: The database server does not report the maximum column name length.")
 
-        table_properties = Encoding_Limited_Naming_Properties.Instance encoding table_limit
-        column_properties = Encoding_Limited_Naming_Properties.Instance encoding column_limit
+        ## Postgres column/table names may be lowercased if unquoted, but when
+           quoted we have full case sensitivity and can distinguish columns
+           `A` from `a` as different.
+           Our generator is supposed to always quote identifiers.
```

Comment:
Could this cause a problem if the user includes raw SQL in a larger query, and expects case insensitivity, in a join for example?